src/Features/Core/Portable/UseThrowExpression/UseThrowExpressionCodeFixProvider.cs

```diff
@@ -54,13 +55,25 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 var throwStatementExpression = root.FindNode(diagnostic.AdditionalLocations[1].SourceSpan);
                 var assignmentValue = root.FindNode(diagnostic.AdditionalLocations[2].SourceSpan);
 
-                // First, remote the if-statement entirely.
+                var text = document.GetTextSynchronously(cancellationToken);
+                var throwLine = text.Lines.GetLineFromPosition(diagnostic.AdditionalLocations[1].SourceSpan.Start);
+
+                var fullThrowStatement = root.FindNode(throwLine.SpanIncludingLineBreak, getInnermostNodeForTie: true);
```

Comment:
don't do this.  you have the span of the node (from .AdditionalLocations).  Find that node, and then use that node to find the statement if it's there.

Reply:
once you have the statement you can grab off the trivia as appropriate and move around.